tests/transports-integrations/tests/integrations/test_anthropic.py

```
@@ -35,12 +35,16 @@
     MULTIPLE_TOOL_CALL_MESSAGES,
     IMAGE_URL,
     BASE64_IMAGE,
+    INVALID_ROLE_MESSAGES,
     WEATHER_TOOL,
     CALCULATOR_TOOL,
+    ALL_TOOLS,
```

Ruff (0.11.9)

`..utils.common.ALL_TOOLS` imported but unused

Remove unused import: `..utils.common.ALL_TOOLS`

(F401)